charts/spring-boot/spring-boot-app-kill/experiment.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    It can target random pods with a Spring Boot application and allows configuring the assaults to inject app-kill
+kind: ChaosExperiment
+metadata:
+  name: spring-boot-app-kill
+  labels:
+    name: spring-boot-app-kill
+    app.kubernetes.io/part-of: litmus
+    app.kubernetes.io/component: chaosexperiment
+    app.kubernetes.io/version: latest
+spec:
+  definition:
+    scope: Namespaced
+    permissions:
+      # Create and monitor the experiment & helper pods
+      - apiGroups: [ "" ]
+        resources: [ "pods" ]
+        verbs: [ "create","delete","get","list","patch","update", "deletecollection" ]
+      # Performs CRUD operations on the events inside chaosengine and chaosresult
+      - apiGroups: [ "" ]
+        resources: [ "events" ]
+        verbs: [ "create","get","list","patch","update" ]
+      # Track and get the runner, experiment, and helper pods log
+      - apiGroups: [ "" ]
+        resources: [ "pods/log" ]
+        verbs: [ "get","list","watch" ]
+      # for creating and managing to execute commands inside target container
+      - apiGroups: [ "" ]
+        resources: [ "pods/exec" ]
+        verbs: [ "get","list","create" ]
```

Comment:
Do we need exec permissions? If not we can remove from all places.

Reply:
Indeed you are right, I fix that.

Reply:
we need this permission for cmd probe. It is common in all experiments